ispyb/sp/mxprocessing.py

```diff
@@ -23,6 +23,7 @@ class MXProcessing(ispyb.interface.processing.IF):
             ("file_name", None),
             ("file_path", None),
             ("file_type", None),
+            ("importance_Rank", None),
```

Comment:
```suggestion
            ("importance_rank", None),
```
should this have an upper case letter? Everything else is lower case only

Reply:
Good point!